src/main/java/org/folio/circulation/support/CqlQuery.java

```diff
@@ -42,6 +41,10 @@
       format("%s==(%s)", indexName, join(" or ", wrapValuesInQuotes(filteredValues))), none()));
   }
 
+  public static Result<CqlQuery> lessThan(String index, Object value) {
```

Comment:
Please can unit tests be written for this

Reply:
Done